### ISSUE TITLE
build error,method FusePanicLogger.BatchForget already declared

### ISSUE THREAD

**Issue by ruanwenfeng** [kahing/goofys#754]
`api/common/panic_logger.go:69:27: method FusePanicLogger.BatchForget already declared at api/common/panic_logger.go:41:27`

https://github.com/kahing/goofys/blob/37cf83563db1bd8aa8d0b02356aacdc3210c26d8/api/common/panic_logger.go#L41

https://github.com/kahing/goofys/blob/37cf83563db1bd8aa8d0b02356aacdc3210c26d8/api/common/panic_logger.go#L69

**Comment by matthewblott** [comment]
Yep. I was trying to build this on an arm server and when I commented out the second `BatchForget` function it built as expected.

**Comment by mechanicker** [comment]
The fix has been merged. IMO, this issue could be closed.
